tests/functional/test_api.py

```diff
@@ -68,3 +69,28 @@ def test_options(self, request):
         request.return_value = MockResponse(200, b'')
 
         rest_request('OPTIONS', 'files/')
+
+
+class SignatureTest(unittest.TestCase):
+    def tearDown(self):
+        conf.signed_uploads = False
+
+    @patch('requests.sessions.Session.request', autospec=True)
+    def test_signature_fields_added(self, request):
+        conf.signed_uploads = True
+
+        request.return_value = MockResponse(200, b'[]')
+
+        uploading_request('GET', '')
+        data = request.call_args[1]['data']
+        self.assertIn('expire', data)
+        self.assertIn('signature', data)
+
+    @patch('requests.sessions.Session.request', autospec=True)
+    def test_signature_fields_scipped(self, request):
```

Comment:
What value of `conf.signed_uploads` should be in this test?

Reply:
default is False.